src/commands/prompt/commands.ts

```diff
@@ -28,6 +33,7 @@ export function processCommand(input: string, context: CommandContext): boolean
     output.outputInfo('- /verbose [on|off]: Enable or disable verbose output');
     output.outputInfo('- /stats [on|off]: Enable or disable displaying of response stats');
     output.outputInfo('- /forget: AI will forget previous messages');
+    output.outputInfo('- /save [file]: Conversation will be saved in txt file');
```

Comment:
```suggestion
    output.outputInfo('- /save [filename]: Conversation will be saved in a text file');
```

Reply:
Also get the actual folder from constant, do not hardcode it

Reply:
Done